e2e/templates/kubelet/kubelet_restart.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+STOP_TIME_IN_SEC=$1
+
+sleep 3; # avoid remote host connection lost before pod execution returns
```

Comment:
Could you elaborate more?

Reply:
Otherwise `connect_get_namespaced_pod_exec` will throw exception during command execution since once kubelet stopped, the connection lost.